packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -26,7 +26,6 @@
   background-color: var(--ColorBackgroundPrimary);
   box-shadow: 0 0 0 0 $color-primary-light;
   color: $white;
-  font-family: $body-font-family;
   font-size: ms(0);
```

Comment:
Isnt this part of box too?

Reply:
It is, but Button needs the ability to override it internally for the small size (and possibly theme it?)

Reply:
If we want these components to be themable, they'll need to maintain these variables in their own code, so they aren't relying on the universal value.

Reply:
Is that work that should happen now or when we're taking on individual component theming?

Reply:
Definitely doable later - we have tickets specifically for that work. We'll be adding `font-family` back in there though, so it might not be worth removing here.